.changeset/long-years-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/connectivity': major
+---
+
+[Compatibility Note] The `Protocol` enum was replaced with a union type. Now, instead of passing `Protocol.HTTPS` pass 'https'.
```

Comment:
```suggestion
[Compatibility Note] The `Protocol` enum was replaced with a string literal union type. Now, instead of passing `Protocol.HTTPS` pass 'https'.
```